Tests/Marketplace/search_and_install_packs.py

```diff
@@ -168,8 +168,79 @@ def search_pack(client: demisto_client, prints_manager: ParallelPrintsManager, p
         lock.release()
 
 
+def install_nightly_packs(client, host, prints_manager, thread_index, packs_to_install, request_timeout=999999):
+    """
+    Install content packs on nightly build.
+    Args:
+        client(demisto_client): The configured client to use.
+        host (str): The server URL.
+        prints_manager (ParallelPrintsManager): Print manager object.
+        thread_index (int): the thread index.
+        packs_to_install (list): A list of the packs to install.
+        request_timeout (int): Timeout settings for the installation request.
+
+    Returns:
+
+    """
+    packs_to_install_str = ', '.join([pack['id'] for pack in packs_to_install])
+    message = 'Installing the following packs in server {}:\n{}'.format(host, packs_to_install_str)
+    prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN, include_timestamp=True)
+    prints_manager.execute_thread_prints(thread_index)
+    # make the pack installation request
+    global PACK_INSTALL
+    PACK_INSTALL = False
+    request_data = {
+        'packs': packs_to_install,
+        'ignoreWarnings': True
+    }
+    while PACK_INSTALL is not True:
+        try:
+            response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                                path='/contentpacks/marketplace/install',
+                                                                                method='POST',
+                                                                                body=request_data,
+                                                                                accept='application/json',
+                                                                                _request_timeout=request_timeout)
+
+            if 200 <= status_code < 300:
```

Comment:
Add a debug log with the response.
The response in this case is a list of dicts with the keys: **id**, **name**, **lastInstallDate**, **updateAvailable**.
Example response:
```
[{'id': 'rasterize', 'currentVersion': '1.0.4', 'name': 'Rasterize', 'lastInstallDate': '2020-10-12T22:13:45.85429826Z', 'updateAvailable': False}]
```

I think we should print the name and the id for each pack